debug/stanza.proj

```diff
@@ -0,0 +1,16 @@
+packages stz-debug/* defined-in "."
+
+package stz-debug/debugger requires :
```

Comment:
Moved C dependencies out from build target and into a `requires` statement.

This allows any program that pulls in these Stanza packages to automatically pull in the right dependencies.